tests/unit/test_main.py

```diff
@@ -354,7 +354,7 @@ def test_load_switch(self, *args):
         self.assertEqual(switch.id, dpid_a)
         self.assertEqual(switch.dpid, dpid_a)
         self.assertTrue(switch.is_enabled())
-        self.assertFalse(switch.is_active())
+        self.assertTrue(switch.is_active())
```

Comment:
@viniarck I believe this should be False, let me explain why: the idea of this method is to load the switch from the storehouse on startup; not necessarily the switch is connected at this point. So that, I believe the correct behavior is the switch be loaded as `active=False` and, as soon as, the switch establishes a connection it becomes `active=True`. Wdyt?

Reply:
Oh, indeed @italovalcy. This was a mistake. Let me fix this. I'll also revert the `lastseen` update on `kytos` https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L678, I didn't realize before that the `get_switch_or_create` was also used this way here, so we'd have to update the `lastseen` on `of_core`  but I ended up putting in the core to be reused since it wouldn't affect the `is_connected`, but it back fired with this unwanted side effect here. 

The `lastseen` probably in the future might need to be revisited if we reuse it one day when we start to support other southbound protocols, since this one here isn't tied to OpenFlow, but this likely wont be a concern/opportunity in the short term

Reply:
@italovalcy I've just pushed the fixes and opened some PRs, here's a summary: 

- I fixed the asserts on this commit 8e262bc050b8633ef3e27d9a3b3c774d829565e5
- The original fix for the `lastseen` [has been done on `of_core` on this PR #41](https://github.com/kytos-ng/of_core/pull/41)
- I reverted the initial `lastseen` fix [on `kytos` on this PR #152](https://github.com/kytos-ng/kytos/pull/152)
- I had to add this temporary commit 7873cd120f9a14029c3e0cf655000074cb12f1a2, on this branch here, just so it could use `kytos`'s PR 152 mentioned above and tests could pass. I'll revert this temporary commit as soon as PR 152 lands

Thanks for your review. I'm glad this was spotted at this stage, having to fix this later on would be messier.

Reply:
I'm glad you were able to quickly find a solution for this behavior, @viniarck! Thank you and congratulations for the very nice job you've done.